iron-image.html

```diff
@@ -396,8 +389,14 @@
       },
 
       _resolveSrc: function(testSrc) {
-        var baseURI = /** @type {string} */(this.ownerDocument.baseURI);
-        return (new URL(Polymer.ResolveUrl.resolveUrl(testSrc, baseURI), baseURI)).href;
+        var resolved = Polymer.ResolveUrl.resolveUrl(testSrc, this.$.baseURIAnchor.href);
```

Comment:
AFAICT, if you have an anchor somewhere with `href="#"`, asking for `.href` will give you the absolute destination, which ends up being whatever the base URI that would apply to that anchor is.